StripeFinancialConnections/StripeFinancialConnections/Source/FinancialConnectionsSheet.swift

```diff
@@ -158,10 +160,23 @@ final public class FinancialConnectionsSheet {
 
         analyticsClient.log(analytic: FinancialConnectionsSheetPresentedAnalytic(clientSecret: self.financialConnectionsSessionClientSecret), apiClient: apiClient)
         let navigationController = hostController!.navigationController
+        present(navigationController, presentingViewController)
+    }
+
+    private func present(_ navigationController: FinancialConnectionsNavigationController, _ presentingViewController: UIViewController) {
+        let toPresent: UIViewController
+        let animated: Bool
         if UIDevice.current.userInterfaceIdiom == .pad {
             navigationController.modalPresentationStyle = .formSheet
+            toPresent = navigationController
+            animated = true
+        } else {
+            wrapperViewController = ModalPresentationWrapperViewController(vc: navigationController)
+            wrapperViewController?.modalPresentationStyle = .overFullScreen
```

Comment:
If you are to have a special view controller (`ModalPresentationWrapperViewController`) that wraps logic for this unique purpose, you could probably also tuck this information in there too `wrapperViewController?.modalPresentationStyle = .overFullScreen`

Reply:
good idea